tests/apis/cassettes/project_api.yaml

```diff
@@ -0,0 +1,5144 @@
+        53, "height": 7}, "content": "98-7654321", "confidence": 0.882, "isModelConfident":
+        true, "pageNumber": 1}], "invoice-1ReceiverAddress": [{"boundingBox": {"top":
+        276, "left": 360, "width": 90, "height": 41}, "content": "Teststr. 13,\n20100
+        Hamburg,\nGermany", "confidence": 0.764, "isModelConfident": true, "pageNumber":
+        1}], "invoice-1ReceiverAddressRecipient": [{"boundingBox": {"top": 260, "left":
+        361, "width": 60, "height": 8}, "content": "Horst Pluto", "confidence": 0.651,
+        "isModelConfident": true, "pageNumber": 1}, {"boundingBox": {"top": 260, "left":
+        361, "width": 60, "height": 8}, "content": "Horst Pluto", "confidence": 1,
+        "isModelConfident": true, "pageNumber": 1}], "invoice-1PaymentTerms": [{"boundingBox":
+        {"top": 138, "left": 444, "width": 32, "height": 7}, "content": "Net 30",
+        "confidence": 0.936, "isModelConfident": true, "pageNumber": 1}]}, "ocrText":
+        []}]}, "outputUiSchema": {"annotations": {"ui:order": ["advanced-custom-1R7Cjb3K1"]},
+        "ui:field": "https://widgets.super.ai/document-extraction@1"}, "settings":
+        {"downloadSettings": {"customFileNameKeys": [], "expandOutputColumns": true,
+        "includeMessages": false, "includeAnnotations": false, "flattenTables": true,
+        "excludeHeaderRow": false, "fieldDelimiter": ",", "quoteValues": false, "sendEmail":
+        true, "saveSettings": false}, "postProcessing": false, "ttlSettings": {"onCompleted":
+        {"ttlInSeconds": 0, "enabled": false}}}, "name": "Test Project", "templateName":
+        "general_document_processor.router", "projectName": "default", "metadata":
+        null, "appParams": {"params": {"documents": [{"documentId": "advanced-custom-1",
+        "documentType": "Advanced Custom", "annotationFields": [{"fieldId": {"data":
+        "r7cjb3k1", "editable": false}, "fieldName": {"editable": true, "data": "string"},
+        "description": {"data": " ", "editable": true}, "required": {"data": false,
+        "editable": true}, "status": {"data": "Active", "editable": true}, "aliases":
+        {"data": [], "editable": true}, "humanInstructions": {"data": " ", "editable":
+        true}, "advices": {"data": [], "editable": true}, "examples": {"data": [],
+        "editable": true}, "antiExamples": {"data": [], "editable": true}, "validationRules":
+        {"data": [], "editable": true}, "options": {"editable": true, "data": {"type":
+        "String", "contentType": "String", "requireAnnotations": {"selection": true,
+        "annotationType": "BoundingBox"}, "applyRedaction": {"selection": false, "redactionType":
+        "Blacken"}, "searchScope": "All", "minLength": 0, "maxLength": 100000}}}],
+        "presetFields": []}], "prompt": {"tableIdentification": {"prompt_prefix":
+        "Act as an advanced table understanding system from documents. You will analyze
```

Comment:
We probably should sanitize the responses here, otherwise we leak prompts?

Reply:
Good catch. I sanitized it for now manually and will shot another PR to do it with pyvcr